backend/api/v1/endpoints/connections.py

```
@@ -0,0 +1,48 @@
+from schemas.connection import ConnectionCreate, ConnectionResponse
+from api.v1.endpoints.auth import get_current_user  # Ensure authentication middleware is implemented
+from models.user import User
+router = APIRouter()
+
+@router.post("/connect", response_model=ConnectionResponse)
+def send_connection(friend_data: ConnectionCreate, db: Session = Depends(get_db), current_user: dict = Depends(get_current_user)) -> ConnectionResponse:
+    try:
+        new_request = send_request(db, current_user["id"], friend_data.friend_id)
+        return new_request
+    except Exception as e:
+        logging.error(f"Connection error: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)